dist/connection-manager.js

```diff
@@ -0,0 +1,231 @@
+                                        connection: self,
+                                        data: e,
+                                        message: msg,
+                                        query: query
+                                    };
+                                    self.handleErrorListeners(querySetWithFilter.errorCallbacks, query, edsEvent);
+                                    self.handleGlobalListener(edsEvent);
+                                }
+                            }
+                        }
+                    }
+                    /*let querySetsAndCallbacksOnce = self.queriesToQuerySetsAndCallbacks.get(query);
+                    if (querySetsAndCallbacksOnce) {
+                        let dataForQuery = data[query];
+                        querySetsAndCallbacksOnce.count += 1;
+                        for (const [qs, callbacks] of querySetsAndCallbacksOnce.qsCb) {
+                            for (let callback of callbacks) {
+                                //console.log(`Execute callback for query: ${query}`)
+                                try {
+                                    let edsEvent: EDSEvent & EDSEventMessage = {
+                                        type: EDSEventType.Message,
+                                        connection: self,
+                                        data: dataForQuery,
+                                        query: query,
+                                        count: querySetsAndCallbacksOnce.count,
+                                        retrieve: querySetsAndCallbacksOnce.initial,
+                                    }
+                                    callback(edsEvent);
+                                    self.handleGlobalListener(edsEvent);
+                                } catch (e) {
+                                    let msg = `Error while handling data for query: ${query}`;
+                                    const edsEvent: EDSEvent & EDSEventError = {
+                                        type: EDSEventType.ClientQueryError,
+                                        connection: self,
+                                        data: e,
+                                        message: msg,
+                                        query: query
+                                    };
+                                    self.handleErrorListenerForMap(querySetsAndCallbacksOnce, query, edsEvent);
+                                    self.handleGlobalListener(edsEvent);
+                                }
+                            }
+                        }
+                        if (querySetsAndCallbacksOnce.once) {
+                            self.queriesToQuerySetsAndCallbacks.delete(query);
+                            //console.log(`Once query deleted from queriesToQuerySetsAndCallbacks: ${query}`);
+                        }
+                    }*/
```

Comment:
Remove unnecessary comments from published documentations
```suggestion
```

Reply:
This file is not using at all, deleted